shared/src/commonMain/kotlin/org/hyperskill/app/main/presentation/AppFeature.kt

```diff
@@ -35,11 +48,14 @@ interface AppFeature {
             val forceUpdate: Boolean = false
         ) : Message
 
-        data class UserAccountStatus(
+        object AppBecomesActive : Message
+
+        data class FetchAppStartupConfigSuccess(
             val profile: Profile,
+            val subscription: Subscription?,
```

Comment:
according to the reducer usages we can pass `subscriptionType: SubscriptionType?`